### PR TITLE
Added inputElement support from react-autowhatever 5.4.0

### PR DESCRIPTION
**Pull request by angelo-v**
This pull request adds the support for a custom input element, that came with react-autowhatever 5.4.0 to react-autosuggest.

### CODE REVIEW

**Comment by angelo-v** [comment]
The bitHound lint result (indentation should be 8) contradicts the result of "npm run lint" (indentation should be 6). Which one is correct?

File: src/Autosuggest.js [hunk 1]
```diff
@@ -440,6 +444,7 @@ class Autosuggest extends Component {
         inputProps={autowhateverInputProps}
         itemProps={this.itemProps}
         theme={theme}
+        inputElement={this.props.inputElement}
```
Comment (moroshko) [inline src/Autosuggest.js]: If I understand correctly, we'll be passing `undefined` here by default, relying on `react-autowhatever` to have an `'input'` fallback.

What do you think about being more explicit by:
* making `inputElement ` required in `Autosuggest.js`
* default `inputElement` to `'input'` in `AutosuggestContainer.js`

?

**Comment by moroshko** [comment]
@angelo-v The lint issue is probably [a bug in `eslint-plugin-react`](https://github.com/yannickcr/eslint-plugin-react/issues/901). I guess we can lock its version to `6.2.2` until it's fixed.

Could you also please update the docs to reflect the changes, and add tests?

File: src/Autosuggest.js (hunk 1 above)
Comment (angelo-v) [inline src/Autosuggest.js]: Sounds fine for me, I adjusted it accordingly

**Comment by moroshko** [comment]
Thanks!

**Comment by moroshko** [comment]
@angelo-v I ended up renaming `inputElement` to `inputComponent` and constraining it to a React Component (cannot be string like `'textarea'`) to keep Autosuggest accessible.

It is available in [v6.1.0](https://github.com/moroshko/react-autosuggest/releases/tag/v6.1.0) now.

**Comment by angelo-v** [comment]
Ok, great! Thanks for merging!

**Pull request merged**